internal/debug/dumpstacks.go

```diff
@@ -0,0 +1,19 @@
+package debug
+
+import "runtime"
+
+func DumpStacks() string {
+	var (
+		buf       []byte
+		stackSize int
+	)
+	bufStartLen := 10240
+
+	for stackSize == len(buf) {
```

Comment:
Does this actually work? The logic seems highly suspect. I would do:

https://github.com/microsoft/hcsshim/blob/0291ac21db715cc43d6068dc18f78b7abed6ab9d/cmd/containerd-shim-runhcs-v1/service.go#L359